types/mongodb/index.d.ts

```diff
@@ -2257,6 +2260,8 @@ export interface GridFSBucketWriteStreamOptions extends WriteConcern {
 
 /** http://mongodb.github.io/node-mongodb-native/3.1/api/ChangeStream.html */
 export class ChangeStream extends Readable {
+    resumeToken: ResumeToken;
```

Comment:
Can you also update the documentation link of ChangeStream to 3.3? (resumeToken was not documented in the 3.1)

Reply:
Will do. Thanks.